visidata/loaders/hdf5.py

```diff
@@ -1,4 +1,4 @@
-from visidata import VisiData, vd, Sheet, Path, Column, ItemColumn, BaseSheet
+from visidata import VisiData, vd, Sheet, Path, Column, ColumnItem, ItemColumn, BaseSheet, anytype
```

Comment:
Nitpick: ItemColumn is the preferred nomenclature; ColumnItem is deprecated.  (All column types are named FooColumn.)